cmd/run.go

```diff
@@ -29,19 +33,48 @@ func Run(cfgFile string) {
 
 func RunWithContext(cfgFile string, ctx context.Context) {
 	c := &refresh.Configuration{}
-	err := c.Load(cfgFile)
-	if err != nil {
-		log.Fatalln(err)
-		os.Exit(-1)
+
+	if err := loadConfig(c, cfgFile); err != nil {
+		if err != ErrConfigNotExist {
+			log.Fatalln(err)
+			os.Exit(-1)
+		} else {
```

Comment:
You don't need the `else` statement. it will exit or go to the log.println

Reply:
You are correct! I'll drop it, one moment.